draft-ietf-tls-tls13.md

```diff
@@ -2351,9 +2359,9 @@ identity.
 Clients MUST verify that the server's selected_identity is within the
 range supplied by the client, that the server selected the cipher
 suite associated with the PSK,
-and that the "key_share", and
-"signature_algorithms" extensions are consistent with the indicated
-ke_modes and auth_modes values. If these values are not consistent,
+and that the "key_share" extension in the ServerHello is consistent
+with the ke_modes offered in the ClientHello psk_key_exchange_modes
+extension.  If these values are not consistent,
```

Comment:
I actually don't think we should be requiring the client to check for KE not being present. It's just an unnecessary check.